<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab_Edit</class>
 <widget class="QWidget" name="Tab_Edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>&amp;Template</string>
     </property>
     <property name="buddy">
      <cstring>combo_template</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="combo_template">
     <item>
      <property name="text">
       <string>Queue Script</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G03 Conformer Opt.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G03 Docking Opt.</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="push_help">
     <property name="text">
      <string>&amp;Help</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QTextEdit" name="edit_edit"/>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>combo_template</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
